src/scripts/blocks/media-aside/DisplayComponent.js

```diff
@@ -43,7 +46,22 @@ class DisplayComponent extends Component {
 
   render() {
     const { attributes, setAttributes } = this.props;
-    const imageUrl = attributes.mediaUrl ? attributes.mediaUrl : 'turnip';
+    const imageUrl = attributes.mediaUrl ? attributes.mediaUrl : '';
+
+    const alignmentOptions = applyFilters('benenson.block.mediaAside.alignmentOptions', [{
+      /* translators: text alignment. for RTL languages, localise as 'Right' */
+      label: __('Left', 'benenson'),
+      value: 'Left',
+    }, {
+      /* translators: text alignment. for RTL languages, localise as 'Left' */
+      label: __('Right', 'benenson'),
+      value: 'Right',
+    }]);
+
+    const classes = classnames('mediaAside', {
+      'mediaAside--alignmentLeft': !attributes.mediaAlignment,
```

Comment:
If you set a default value in the attribute declaration, you shouldn't need this duplicate class name.